datafusion/core/src/bin/print_functions_docs.rs

```diff
@@ -158,6 +158,23 @@ fn print_docs(
                 unreachable!()
             };
 
+            // let attr_text = documentation.to_doc_attribute();
+            //
+            // let file_path = format!("{}.txt", name);
+            // if std::path::Path::new(&file_path).exists() {
+            //     std::fs::remove_file(&file_path).unwrap();
```

Comment:
I'm thinking of leaving it for now as this part generates text files with attribute text so its easier to do a migration, semi automatic way instead of manual

Reply:
I would just add a small comment noting this is useful for migration purposes.

Reply:
I agree a comment explaining what this is used for would be most helpful